engine/collection/compliance/engine.go

```diff
@@ -505,23 +502,20 @@ func (e *Engine) BroadcastProposalWithDelay(header *flow.Header, delay time.Dura
 		}
 
 		err = e.con.Publish(msg, recipients.NodeIDs()...)
-		if errors.Is(err, network.EmptyTargetList) {
-			return
-		}
 		if err != nil {
-			log.Error().Err(err).Msg("could not broadcast proposal")
-			return
+			if errors.Is(err, network.EmptyTargetList) {
+				return
+			}
+			log.Err(err).Msg("could not send proposal message")
 		}
 
-		log.Info().Msg("cluster proposal was broadcast")
```

Comment:
I think this log is still useful

Reply:
👍 agree, this was a mistake - I re-added the log